tests/integration-tests/conftest_networking.py

```diff
@@ -223,6 +223,9 @@ def get_availability_zones(region, credential):
 def get_az_setup_for_region(region: str, credential: list):
     """Return a default AZ ID and its name, the list of all AZ IDs and names."""
     az_id_to_az_name_map = get_az_id_to_az_name_map(region, credential)
+    if "us-isob" in region:
```

Comment:
We should specify the precise region: us-isob-east-1. What if at some point there will be another region with prefix us-isob-WHATEVER?

Reply:
Will make change